core/templates/dev/head/pages/exploration_editor/exploration_metadata_modal_directive.html

```diff
@@ -6,11 +6,10 @@
   <p>
     Just a few more things before you publish:
   </p>
-  <p ng-if="requireTitleToBeSpecified">
-    Add a title: what should your exploration be called?
-    <input type="text" class="form-control" ng-model="explorationTitleService.displayed"
-           placeholder="Music in the 19th Century" maxlength="40">
-  </p>
+  <title-field
```

Comment:
Perhaps exploration-title-editor is a better name, and you'd store it in exploration_title_editor_directive.html